tests/integration/enum-backed-with-cases-warning/input/index.rst

```diff
@@ -0,0 +1,30 @@
+===================
+PHP Enum with Const
+===================
+
+..  php:enum:: RedSuit : string
```

Comment:
Folder has "with cases warning" - but I assume, there will be also an enum warning (string defined with `: string` and with `:type:`

So the folder name (and therefore the heading) should be adjusted. Or a separate test should be used.

Reply:
renamed both folder and headline